eclass/texlive-common.eclass

```diff
@@ -35,10 +33,10 @@ TEXMF_VAR_PATH=/var/lib/texmf
 
 texlive-common_handle_config_files() {
 	# Handle config files properly
-	[ -d "${ED}${TEXMF_PATH}" ] || return
-	cd "${ED}${TEXMF_PATH}"
-	for f in $(find . -name '*.cnf' -type f -o -name '*.cfg' -type f | sed -e "s:\./::g") ; do
-		if [ "${f#*config}" != "${f}" -o "${f#doc}" != "${f}"  -o "${f#source}"	!= "${f}" -o "${f#tex}" != "${f}" ] ; then
+	[[ -d ${ED}${TEXMF_PATH} ]] || return
+	cd "${ED}${TEXMF_PATH}" || die
+	find . -name '*.cnf' -type f -o -name '*.cfg' -type f | sed -e "s:\./::g" | while read f; do
+		if [[ ${f#*config} != ${f} || ${f#doc} != ${f} || ${f#source} != ${f} || ${f#tex} != ${f} ]] ; then
```

Comment:
Did you mean:

    [[ ${f} == *config ]]

? Or sth else because I'm not really sure. Don't use this POSIX-ish comparison crap that's completely unreadable. Maybe even `case` would be shorter here.